src/Raven.Client/Documents/Smuggler/DatabaseSmugglerImportOptions.cs

```diff
@@ -14,9 +14,14 @@ public DatabaseSmugglerImportOptions(DatabaseSmugglerOptions options)
             RemoveAnalyzers = options.RemoveAnalyzers;
             TransformScript = options.TransformScript;
         }
+
+        public bool FromCsv { get; set; }
```

Comment:
We are not using this anywhere in the server, which means that this is a good indication that it should be handled differently.

I suggest adding ImportCsvAsync method or creating a command only for this test

Reply:
For now I've removed this and moved the command to tests. I'm not sure if we want to have this functionality in client-smuggler, but if we do it needs to have a separate method with separate options since it does not use any of the Import options at all. Can be done in next iteration.